app/services/fetch_github_pull_request_data_service.rb

```diff
@@ -0,0 +1,100 @@
+class FetchGithubPullRequestDataService
```

Comment:
Naming: don't add service at the end, `FetchGithubPullRequestData` is enough

Reply:
renamed